adapter/ipListChecker/cfg.proto

```diff
@@ -0,0 +1,32 @@
+// Copyright 2016 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package ipListChecker;
+
+message Config {
+  // ProviderURL is the URL where to find the list to check against
+	string provider_url = 1;
+
+	// RefreshInterval determines how often the provider is polled for
+	// an updated list in seconds
+	int64 refresh_interval = 2;
```

Comment:
Using the Duration type?

At the very least, indicate the unit (seconds, nanos?)